react/css/src/digitv2/components/mobileNumberV2.scss

```
@@ -62,4 +62,94 @@
+    .digit-dropdown-select-active {
+      height: var(--digitv2-spacers-spacer10);
+      border-top-right-radius: 0 !important;
+      border-bottom-right-radius: 0 !important;
+      border-right: none !important;
+      border-top-left-radius: var(--digitv2-borderRadius-radius2);
+      border-bottom-left-radius: var(--digitv2-borderRadius-radius2);
+    }
+
+    .digit-dropdown-employee-select-wrap--elipses {
+      @apply font-bold;
+      font-size: var(--digitv2-spacers-spacer4);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)